src/daemon/bitcoind/actions.rs

```diff
@@ -323,6 +324,30 @@ fn mark_confirmed_spends(
     Ok(())
 }
 
+fn mark_confirmed_cancels(
+    revaultd: &Arc<RwLock<RevaultD>>,
+    bitcoind: &BitcoinD,
+) -> Result<(), BitcoindError> {
+    let db_path = revaultd.read().unwrap().db_file();
+
+    for (db_vault, cancel_tx) in db_canceling_vaults(&db_path)? {
+        let cancel_tx = cancel_tx.into_psbt().extract_tx();
```

Comment:
I think `txid()` can be used here too. Can you include the fix in be83f4f1b6d90e92d62ad113dd2c2563b886f623?

Reply:
Right, good catch